overlord/snapstate/handlers.go

```diff
@@ -1894,13 +1894,17 @@ func (m *SnapManager) doClearSnapData(t *state.Task, _ *tomb.Tomb) error {
 	}
 
 	if err = m.backend.RemoveSnapData(info); err != nil {
-		return err
+		st.Lock()
+		t.Logf("Cannot remove snap data: %v", err)
```

Comment:
This seems like a perfect place to warn instead.

Reply:
I thought a bit and I think I agree,  though in practice I thing we should do both